nixos/tests/wordpress.nix

```diff
@@ -23,19 +23,26 @@ import ./make-test.nix ({ pkgs, ... }:
     };
 
   testScript = ''
-    startAll;
-
-    $machine->waitForUnit("httpd");
-    $machine->waitForUnit("phpfpm-wordpress-site1.local");
-    $machine->waitForUnit("phpfpm-wordpress-site2.local");
-
-    $machine->succeed("curl -L site1.local | grep 'Welcome to the famous'");
-    $machine->succeed("curl -L site2.local | grep 'Welcome to the famous'");
-
-    $machine->succeed("systemctl --no-pager show wordpress-init-site1.local.service | grep 'ExecStart=.*status=0'");
-    $machine->succeed("systemctl --no-pager show wordpress-init-site2.local.service | grep 'ExecStart=.*status=0'");
-    $machine->succeed("grep -E '^define.*NONCE_SALT.{64,};\$' /var/lib/wordpress/site1.local/secret-keys.php");
-    $machine->succeed("grep -E '^define.*NONCE_SALT.{64,};\$' /var/lib/wordpress/site2.local/secret-keys.php");
+    start_all()
+
+    machine.wait_for_unit("httpd")
+    machine.wait_for_unit("phpfpm-wordpress-site1.local")
+    machine.wait_for_unit("phpfpm-wordpress-site2.local")
+
+    machine.succeed("curl -L site1.local | grep 'Welcome to the famous'")
+    machine.succeed("curl -L site2.local | grep 'Welcome to the famous'")
+
+    machine.succeed(
+        "systemctl --no-pager show wordpress-init-site1.local.service | grep 'ExecStart=.*status=0'"
+    )
+    machine.succeed(
+        "systemctl --no-pager show wordpress-init-site2.local.service | grep 'ExecStart=.*status=0'"
+    )
+    machine.succeed(
+        "grep -E '^define.*NONCE_SALT.{64,};\$' /var/lib/wordpress/site1.local/secret-keys.php"
+    )
+    machine.succeed(
+        "grep -E '^define.*NONCE_SALT.{64,};\$' /var/lib/wordpress/site2.local/secret-keys.php"
+    )
```

Comment:
The only thing that defers between these two commands is the service name being `1` or `2` or this directory having `site{number}`.

Reply:
True. something like this might be nice:

```python
for nr in 1, 2:
    machine.wait_for_unit(f"phpfpm-wordpress-site{nr}.local")
    machine.succeed(
        f"curl -L site{nr}.local | grep 'Welcome to the famous'",
        ...
    )
```